### ISSUE TITLE
Transparent/customisable cover background

### ISSUE THREAD

**Issue by Berkmann18** [Wlada/vue-carousel-3d#145]
It would be neat to have the option to set the cover slides to transparent or a custom colour so images with transparency wouldn't look weird.

**Comment by jsuria** [comment]
I agree. Though setting the transparency via CSS also works as well.

**Comment by Berkmann18** [comment]
@jsuria That's what I've done for now.

**Comment by Wlada** [comment]
Answered — slide background is CSS; override with `background-color: transparent` on `.carousel-3d-slide`.
